sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -4,316 +4,689 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------
 
-from typing import List, Any, Optional, TYPE_CHECKING
+from typing import Dict, List, Any, Optional, TYPE_CHECKING, Tuple, Union
 
 from azure.core import PipelineClient
+from cryptography.hazmat.primitives import serialization
 from six import python_2_unicode_compatible
 
 if TYPE_CHECKING:
```

Comment:
I think it's mostly stylistic if pylint doesn't complain, but this `if TYPE_CHECKING:` block usually comes after regular imports ([example](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/aio/_client.py#L16))

Reply:
Actually, I don't think pylint is running normally in CI here. To check pylint using the repo's style rules, you can run `pylint --rcfile=pylintrc {file or directory you want to check}` from the very top of the repo. 

So if your current directory is `...\azure-sdk-for-python\sdk\attestation\azure-security-attestation`, you can run `pylint --rcfile=..\..\..\pylintrc azure\security\attestation\_administration_client.py` to evaluate this file. There are quite a few errors (mostly docstring lines being too long, but some legitimate ones) that aren't being caught by the pylint check in CI.

@scbedd does this seem to be the case, or am I missing something?

Reply:
Yeah, that... After an hours work, I'm up to 8.74, from 7.4 when I started. Next push will have all the fixes.